docs/contribution-guide/1-overview.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2024-05-10
+category: document
+author: Revanth Rao Thopucherla
+order: 1
+---
+
+# Contributing to Product Documentation
+
+Thank you for your interest in contributing to our product documentation! We welcome contributions from everyone to help improve the quality and clarity of our docs. This document outlines the guidelines and processes for contributing.
+
+## Ways to Contribute
+
+1. **Report/Suggest Enhancements**: If you find any errors, typos, or areas that need improvement or ideas for new topics, examples, or improvements to existing content, feel free to [open an issue](https://github.com/jangala-dev/product-docs/issues) and describe your suggestions/problem. We'll review your issue and address it as soon as possible.
```

Comment:
I would swap Report/Suggest Enhancements to 2 and submit pull requests to 1 since this is the way our documentation is ordered

Reply:
done